config/cesm/config_grids.xml

```diff
@@ -450,35 +466,67 @@
       <mask>tn0.25v3</mask>
     </model_grid>
 
+    <model_grid alias="f09_g16_gris4" compset="_CISM">
```

Comment:
Are we really still supporting g16? I did not think so. Could you check if moving forwards we just remove this resolution. I am wondering at this point if it should totally be removed from config_grids.xml for cesm. At any rate - it certainly does not make sense to add a new item for CISM for it if we are no longer supporting it.